lib/stripe/file.rb

```diff
@@ -1,11 +1,12 @@
 # frozen_string_literal: true
 
 module Stripe
-  class FileUpload < APIResource
+  class File < APIResource
     extend Stripe::APIOperations::Create
     extend Stripe::APIOperations::List
 
-    OBJECT_NAME = "file_upload".freeze
+    OBJECT_NAME = "file".freeze
+    ALT_OBJECT_NAME = "file_upload".freeze
```

Comment:
Mind just adding a bit of a comment here explaining that these used to be two distinct objects that were then unified into one?

Reply:
Also, sorry for the super nit, but just so this sorts nicely, do you think we could go `OBJECT_NAME_ALT`? :)